src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -478,6 +478,12 @@
   <data name="Arg_MustBeChar" xml:space="preserve">
     <value>Object must be of type Char.</value>
   </data>
+  <data name="Arg_MustBeDateOnly" xml:space="preserve">
+    <value>Object must be of type DateOnly.</value>
```

Comment:
❔ Does this repository generally add comments to indicate locked strings? In this case, it may not be obvious to localizers that `DateOnly` is a fixed string, but `Object` is not.

Reply:
Thanks. I'll try to use that, or I can pass the name as a parameter to the message.